src/shell/CMakeLists.txt

```diff
@@ -104,12 +104,26 @@ endif()
 
 # LEAN TESTS
 file(GLOB LEANTESTS "${LEAN_SOURCE_DIR}/../tests/lean/*.lean")
+add_test(NAME leantest_clear_all
+        WORKING_DIRECTORY "${LEAN_SOURCE_DIR}/../tests/lean"
+        COMMAND bash -c "rm *.test_suite.out *.status || true")
+set_tests_properties(leantest_clear_all PROPERTIES FIXTURES_SETUP AllFiles)
```

Comment:
Note: `FIXTURES_SETUP` is cmake >= 3.7. In older cmakes,running a leantest/leanruntest t will show an outdated result in the unlikely sequence
* run `lean(run)test_all` but not t
* make a change that changes t's result
* run t but not `lean(run)test_all`